docs/OV_Runtime_UG/multi_device.md

```diff
@@ -1,39 +1,41 @@
+* increase in performance stability due to multiple devices sharing inference workload.
 
-* Improved throughput from using multiple devices (compared to single-device execution)
-* More consistent performance, since the devices share the inference burden (if one device is too busy, another can take more of the load)
+What is important, the Multi-Device mode does not change tha application logic, so it does not require you to explicitly compile the model on every device or create and balance inference requests. From the application point of view, this is just another plugin that handles the actual machinery.
 
-Note that with Multi-Device the application logic is left unchanged, so you don't need to explicitly compile the model on every device, create and balance the inference requests and so on. From the application point of view, this is just another device that handles the actual machinery. The only thing that is required to leverage performance is to provide the multi-device (and hence the underlying devices) with enough inference requests to process. For example, if you were processing 4 cameras on the CPU (with 4 inference requests), it might be desirable to process more cameras (with more requests in flight) to keep CPU and GPU busy via Multi-Device.
+Note that performance increase in this mode comes from utilizing multiple devices at once. This means that you need to provide the devices with enough inference requests to keep them busy or you will not benefit much from using MULTI.
 
-The setup of Multi-Device can be described in three major steps:
 
-1. Prepare configure for each device. 
-2. Compile the model on the Multi-Device plugin created on top of a (prioritized) list of the configured devices with the configure prepared in step one.
-3. As with any other CompiledModel call (resulting from `compile_model`), you create as many requests as needed to saturate the devices.
+## Using the Multi-Device Mode 
 
-These steps are covered below in detail.
+Following the OpenVINO™ naming convention, the Multi-Device mode is assigned the label of “MULTI.” The only configuration option available for it is a prioritized list of devices to use:
 
-### Defining and Configuring the Multi-Device Plugin
+@sphinxdirective
 
-Following the OpenVINO™ convention of labeling devices, the Multi-Device plugin uses the name "MULTI". The only configuration option for the Multi-Device plugin is a prioritized list of devices to use:
++---------------------------+---------------------------------+-----------------------------------------------------------+
+| Property                  | Property values                 | Description                                               |
++===========================+=================================+===========================================================+
+| <device candidate list>   | | AUTO: <device names>          || Specifies the devices available for selection.           |
```

Comment:
unlike AUTO, these are NOT candidates: if loading to any device from the list fails, the entire MULTI would fail as well